tests/unit/OptionsListUtilsTest.js

```diff
@@ -296,7 +296,7 @@ describe('OptionsListUtils', () => {
 
     it('getSearchOptions()', () => {
         // When we filter in the Search view without providing a searchValue
-        let results = OptionsListUtils.getSearchOptions(REPORTS, PERSONAL_DETAILS, '');
+        let results = OptionsListUtils.getSearchOptions(REPORTS, PERSONAL_DETAILS, '', [CONST.BETAS.ALL]);
```

Comment:
I'm a bit confused, why are we passing `CONST.BETAS.ALL` here now, and how does this relate to this PR / the other changes?